server/issue_handler.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+func (s *Server) issueEventHandler(w http.ResponseWriter, r *http.Request) {
+	ctx, cancel := context.WithTimeout(context.Background(), defaultRequestTimeout*time.Second)
+	defer cancel()
+
+	event, err := PullRequestEventFromJSON(r.Body)
+	if err != nil {
+		mlog.Error("could not parse issue event", mlog.Err(err))
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	parts := strings.Split(event.Issue.GetHTMLURL(), "/")
+	if len(parts) < 4 {
+		mlog.Error("incorrect pattern for issue url")
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	mlog.Info("handle issue event", mlog.String("repoUrl", *event.Issue.HTMLURL), mlog.String("Action", event.Action), mlog.Int("PRNumber", event.PRNumber))
+	issue, err := s.GetIssueFromGithub(ctx, parts[len(parts)-4], parts[len(parts)-3], event.Issue)
```

Comment:
Can we add a comment to clarify what does `parts[len(parts)-4]` and `parts[len(parts)-3]` signify?